doc/manual/manual.tex

```diff
@@ -969,6 +969,31 @@ \subsubsection{Coefficient self-consistency}
   compressibility for each of the different approximations. The conversion between isothermal
   and isentropic compressibilities is given in ~\eqref{eq:mm_isentropic_compressibility}.}
 
+
+\subsubsection{Coefficient averaging}
+In multiphase rocks, or multirock areas in convection simulations, properties must be averaged.
```

Comment:
would you be willing to add a half sentence of the form "...because the length scales at which the rock types vary is far smaller than the resolution of the mesh. As a consequence, we need to use ``effective coefficients'', i.e., coefficients that do not correspond to any particular rock, but that lead to a macroscopic response that is a good match to the response of the correct, but unresolvable mixture of rocks."

Reply:
Sure. Done!